mappings/net/minecraft/client/render/chunk/ChunkRenderer.mapping

```diff
@@ -44,14 +44,16 @@ CLASS net/minecraft/class_846 net/minecraft/client/render/chunk/ChunkRenderer
 		FIELD field_4459 data Ljava/util/concurrent/atomic/AtomicReference;
 		FIELD field_4461 sortTask Lnet/minecraft/class_846$class_851$class_4579;
 		FIELD field_4462 buffers Ljava/util/Map;
-		FIELD field_4463 rebuildOnClientThread Z
-		FIELD field_4464 rebuildScheduled Z
+		FIELD field_4463 needsImportantRebuild Z
```

Comment:
What does "important" mean?

Reply:
The rebuild task has higher priority, and it will rebuild on the client thread if the option to rebuild near on client thread is enabled. It's used for blocks placed by a player, for example (it's important for those to show up immediately).